docs/en/ingest-management/release-notes/release-notes-8.8.asciidoc

```diff
@@ -0,0 +1,226 @@
+* Added support for the Cloud Defend application to {agent} {fleet-server-pull}2477[#2477]
+* Disabled signature validation in {agent} so that only {endpoint-sec} validates policies and actions {fleet-server-pull}2562[#2562]
+
+{fleet-server}::
+* Replaced upgrade expiration and `minimum_execution_duration` with rollout_duration_seconds` {fleet-server-pull}2243[#2243]
+* Added a `poll_timeout` attribute to check in requests that the client can use to inform {fleet-server} of how long the client will hold the polling connection open for {fleet-server-pull}2491[#2491] {fleet-server-issue}2337[#2337]
+* Added a `memory_limit` configuration setting to help prevent OOM errors {fleet-server-pull}2514[#2514]
+
+{agent}::
+* Make download of {agent} upgrade artifacts asynchronous during Fleet-managed upgrade {agent-pull}2205[#2205] {agent-issue}1706[#1706]
+* Make the language used in CLI commands more consistent {fleet-server-pull}2496[#2496]
+
+[discrete]
+[[bug-fixes-8.8.0]]
+=== Bug fixes
+
+{fleet}::
+* Fixes package license check to use new `conditions.elastic.subscription` field {kibana-pull}154831[#154831]
+* Fixes the OpenAPI spec from `/agent/upload` to `/agent/uploads` for Agent uploads API {kibana-pull}151722[#151722]
+
+{fleet-server}::
+* Filter out unused `UPDATE_TAGS` and `FORCE_UNENROLL` actions from being delivered to {agent} {fleet-server-pull}2200[#2200]
+* Ignore the `unenroll_timeout` field on agent policies as it has been replaced by a configurable inactivity timeout {fleet-server-pull}2096[#2096] {fleet-server-issue}2063[#2063]
+* Fixed {fleet-server} discarding duplicate `server` keys input when creating configuration from a policy {fleet-server-pull}2354[#2354] {fleet-server-issue}2303[#2303]
+* {fleet-server} will no longer restart subsystems like API listeners and the {es} client when the log level changes {fleet-server-pull}2454[#2454] {fleet-server-issue}2453[#2453]
+
+{agent}::
+* Fixed the formatting of system metricsets in example {agent} configuration file {agent-pull}2338[#2338]
+* Fixed the parsing of paths from the `container-paths.yml` file {agent-pull}2340[#2340]
+* Added a check to ensure that {agent} was bootstrapped with the `--fleet-server-*` options {agent-pull}2505[#2505] {agent-issue}2170[#2170]
+* Fixed an issue where inspect and diagnostics didn't include the local {agent} configuration {agent-pull}2529[#2529] {agent-issue}2390[#2390]
+* Fixed a bug that caused heap profiles captured in the agent diagnostics to be unusable {agent-pull}2549[#2549] {agent-issue}2530[#2530]
+* Fix an issue that occurs when specifing a `FLEET_SERVER_SERVICE_TOKEN_PATH` with the agent running in a Docker container where both the token value and path are passed in the enroll section of the agent setup {agent-pull}2576[#2576]
+* Created an install marker file during upgrade from versions prior to 8.8.0 to help support the new custom base path feature {agent-pull}2655[#2655] {agent-issue}2645[#2645]
```

Comment:
```suggestion
```

We hid this feature so we don't need this changelog entry yet. This is also a fix for a bug that was never released.